backend/Cargo.toml

```
@@ -9,6 +9,7 @@ repository = { workspace = true }
 
 [features]
 halo2 = ["dep:powdr-halo2"]
+avx512 = ["starky/avx512"]
```

```suggestion
starky-avx512 = ["starky/avx512"]
```

I think this name would be better because it signals that this is important for starky, and it's not powdr that's using avx itself